server/src/main/java/org/elasticsearch/cluster/InternalClusterInfoService.java

```diff
@@ -537,6 +540,10 @@ private boolean shouldRefresh() {
 
     @Override
     public ClusterInfo getClusterInfo() {
+        return currentClusterInfo;
+    }
+
+    private void updateAndGetCurrentClusterInfo() {
```

Comment:
The method name here hints that it should return the cluster info? That would seem nice to do, but I'd also be fine to just call it `updateClusterInfo`

Reply:
Yeah it was an oversight and was meant to return the value. Fixed in 001af0d21cc5128df1356cd45421cb33fedfb18b